velox/tpcds/gen/utils/CMakeLists.txt

```
@@ -0,0 +1,16 @@
+# Copyright (c) Facebook, Inc. and its affiliates.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+velox_add_library(velox_tpcds_append_info OBJECT append_info-c.cpp)
+velox_link_libraries(velox_tpcds_append_info Folly::folly xsimd)
```

```suggestion
velox_add_library(velox_tpcds_append_info append_info-c.cpp)
velox_link_libraries(velox_tpcds_append_info PRIVATE velox_vector fmt::fmt)
```
Object libraries can be have weirdly. Please only link what is actually used.